### PR TITLE
README: add CNCF announcement and logo.

### PR DESCRIPTION
**Pull request by c4pt0r**
## What have you changed? (mandatory)
Modify README.md, add CNCF announcement and logo

Please explain **IN DETAIL** what the changes are in this PR and why they are needed:
- Summarize your change (mandatory)
- How does this PR work? Need a brief introduction for the changed logic (optional)
- Separately describe each logical change and avoid lazy messages (optional)
- Describe any limitations of the current code (optional)

Please **NOTE** that:
- Do not assume reviewers understand the original issue

## What are the type of the changes? (mandatory)

The currently defined types are listed below, please pick one of the types for this PR by removing the others:
- New feature (non-breaking change which adds functionality)
- Improvement (non-breaking change which is an improvement to an existing feature)
- Bug fix (non-breaking change which fixes an issue)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)

## How has this PR been tested? (mandatory)

Please describe the tests that you ran to verify your changes. Have you finished unit tests, integration tests, or manual tests? What additional tests would give you greater confidence in this change?

## Does this PR affect documentation (docs/docs-cn) update? (mandatory)

If there is document change, please file a PR in ([docs](https://github.com/pingcap/docs) or [docs-cn](https://github.com/pingcap/docs-cn)) and add the PR number here.

## Does this PR affect tidb-ansible update? (mandatory)

If there is a configuration or metrics change, please file a PR in [tidb-ansible](https://github.com/pingcap/tidb-ansible), and add the PR number here.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,17 @@
-# TiKV
+![tikv_logo](images/logo.png)
 
 [![Build Status](https://circleci.com/gh/tikv/tikv.svg?style=shield&circle-token=36bab0a8e43edb0941b31c38557d2d9d0d58f708)](https://circleci.com/gh/tikv/tikv) [![Coverage Status](https://codecov.io/gh/tikv/tikv/branch/master/graph/badge.svg)](https://codecov.io/gh/tikv/tikv) ![GitHub release](https://img.shields.io/github/release/tikv/tikv.svg)
 
+
+TiKV is an open source distributed Key-Value storage system; Unlike other traditional NoSQL systems, TiKV not only provides classical Key-Value style APIs, but also ACID semantic APIs.
```
Comment (siddontang) [inline README.md]: seem these are duplicated with following introductions.

**Comment by siddontang** [comment]
PTAL @queenypingcap

File: README.md (hunk 1 above)
Comment (c4pt0r) [reply README.md]: The reason why I put a short description here is I think maybe some engineers can't have the patience to read the full description below, so a '1-sentence' intro seems a good idea.

Comment (queenypingcap) [reply README.md]: I agree with @c4pt0r that 1-sentence intro is a good idea, but let's avoid duplicating information. Let's edit and move [the introduction](https://github.com/tikv/tikv/pull/3795#diff-04c6e90faac2675aa89e2176d2eec7d8R15) here.

Comment (queenypingcap) [reply README.md]: TiKV ("Ti" stands for Titanium) is an open source distributed transactional key-value database. Unlike other tranditional NoSQL systems, TiKV not only provides classical Key-Value APIs, but also transactional APIs with ACID compliance. Built in Rust and powered by Rust, TiKV was originally created to complement [TiDB](https://github.com/pingcap/tidb), a distributed HTAP database compatible with the MySQL protocol.

File: README.md [hunk 2]
```diff
@@ -1,7 +1,17 @@
...
+
+The design of TiKV is deeply influenced by some great distributed systems by Google, including BigTable, Spanner and Percolator, combined some of the latest achievements in academia in recent year, e.g. Raft consensus algorithm;
+![cncf_logo](images/cncf.png)
+
+TiKV is now hosted by the Cloud Native Computing Foundation (CNCF), If you are a company that wants to help shape the evolution of technologies that are container-packaged, dynamically-scheduled and microservices-oriented, consider joining the CNCF. For details about who's involved and how TiKV plays a role, read the CNCF [announcement](https://www.cncf.io/blog/2018/08/28/cncf-to-host-tikv-in-the-sandbox/).
+
+---
+
 TiKV ("Ti" stands for Titanium) is a distributed transactional key-value database, originally created to complement [TiDB](https://github.com/pingcap/tidb), a distributed HTAP database compatible with the MySQL protocol. TiKV is built in Rust and powered by Raft, and was inspired by the design of Google Spanner and HBase, but without dependency on any specific distributed file system.
```
Comment (queenypingcap) [inline README.md]: Please remove this para.

File: README.md [hunk 3]
```diff
@@ -1,7 +1,17 @@
...
 
+
+TiKV is an open source distributed Key-Value storage system; Unlike other traditional NoSQL systems, TiKV not only provides classical Key-Value style APIs, but also ACID semantic APIs.
+
+The design of TiKV is deeply influenced by some great distributed systems by Google, including BigTable, Spanner and Percolator, combined some of the latest achievements in academia in recent year, e.g. Raft consensus algorithm;
+![cncf_logo](images/cncf.png)
+
+TiKV is now hosted by the Cloud Native Computing Foundation (CNCF), If you are a company that wants to help shape the evolution of technologies that are container-packaged, dynamically-scheduled and microservices-oriented, consider joining the CNCF. For details about who's involved and how TiKV plays a role, read the CNCF [announcement](https://www.cncf.io/blog/2018/08/28/cncf-to-host-tikv-in-the-sandbox/).
```
Comment (queenypingcap) [inline README.md]: Cloud Native Computing Foundation (CNCF) -> [Cloud Native Computing Foundation](https://cncf.io/)  (CNCF)

File: README.md [hunk 4]
```diff
@@ -1,7 +1,17 @@
-# TiKV
+![tikv_logo](images/logo.png)
 
 [![Build Status](https://circleci.com/gh/tikv/tikv.svg?style=shield&circle-token=36bab0a8e43edb0941b31c38557d2d9d0d58f708)](https://circleci.com/gh/tikv/tikv) [![Coverage Status](https://codecov.io/gh/tikv/tikv/branch/master/graph/badge.svg)](https://codecov.io/gh/tikv/tikv) ![GitHub release](https://img.shields.io/github/release/tikv/tikv.svg)
 
+
+TiKV is an open source distributed Key-Value storage system; Unlike other traditional NoSQL systems, TiKV not only provides classical Key-Value style APIs, but also ACID semantic APIs.
+
+The design of TiKV is deeply influenced by some great distributed systems by Google, including BigTable, Spanner and Percolator, combined some of the latest achievements in academia in recent year, e.g. Raft consensus algorithm;
```
Comment (queenypingcap) [inline README.md]: -> The design of TiKV is inspired by some great distributed systems by Google, such as BigTable, Spanner, and Percolator, and some of the latest achievements in the academia in recent years, such as the Raft consensus algorithm.

File: README.md [hunk 5]
```diff
@@ -1,7 +1,17 @@
-# TiKV
+![tikv_logo](images/logo.png)
```
Comment (Hoverbear) [inline README.md]: Please use a higher quality image, preferring an SVG. This is noticeably pixelated on high DPI screens.

**Comment by c4pt0r** [comment]
Nice comments. I'll resolve these.

**Comment by lilin90** [comment]
I'll close this PR because of the DCO error. The content update is involved in https://github.com/tikv/tikv/pull/3812, with comments in this PR addressed.

**Pull request closed without merging**